todo-app/public/index.html

```diff
@@ -10,34 +10,13 @@
       content="Web site created using create-react-app"
```

Comment:
this is a comment regarding how you name commits. 
`(this commit will)` part is only for you to know how to phrase a commit message and should not actually be included in the commit message :)